lib/booking/repositories/booking_repository.dart

```diff
@@ -4,37 +4,37 @@ import 'package:myecl/tools/repository/repository.dart';
 class BookingRepository extends Repository {
   @override
   // ignore: overridden_fields
-  final ext = 'bdebooking/bookings';
+  final ext = 'booking/bookings/';
 
   Future<List<Booking>> getBookingList() async {
-    return List<Booking>.from(
-        (await getList()).map((x) => Booking.fromJson(x)));
+    return List<Booking>.from((await getList(suffix: "user/managers"))
+        .map((x) => Booking.fromJson(x)));
   }
 
   Future<Booking> createBooking(Booking booking) async {
     return Booking.fromJson(await create(booking.toJson()));
```

Comment:
I added a trailing slash too in Hyperion.
I added the trailing slash because there could be issues with the endpoint booking/bookings/{booking_id}. If there is another endpoint like booking/bookings/managers which returns the bookings a user can manage it might not find it because it will try to access the endpoint booking/bookings/{booking_id} with booking_id=managers.

Reply:
I forgot to mention that I removed the trailing slash because I didn't need it anymore with the urls I kept in the end. I just wanted to justify why I added it in the first place.